src/components/DiagnosticsBody.vue

```diff
@@ -51,6 +53,7 @@ export default Vue.extend({
         },
       ],
       hideMuted: false,
+      timeStamp: dayjs(),
```

Comment:
It might be better to call this `now` or `currentTime`; `timeStamp` sounds like it would be mostly constant, not updated every second.